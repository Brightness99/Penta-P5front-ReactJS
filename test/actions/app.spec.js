import * as Actions from 'actions';

describe('App', () => {
  it('goTo should create an action to navigate with react-router', () => {
    expect([Actions.goTo('/destination')]).toMatchSnapshot();
  });

  it('showAlert should return an action', () => {
    expect(Actions.showAlert('success', 'Alright!', false)).toMatchSnapshot();
  });

  it('hideAlert should return an action', () => {
    expect(Actions.hideAlert()).toMatchSnapshot();
  });
});
