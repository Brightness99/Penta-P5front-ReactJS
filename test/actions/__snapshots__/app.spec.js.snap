// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App goTo should create an action to navigate with react-router 1`] = `
Array [
  Object {
    "action": "PUSH",
    "location": Object {
      "pathname": "/destination",
      "search": undefined,
      "state": undefined,
    },
    "type": "@@router/LOCATION_CHANGE",
  },
]
`;

exports[`App hideAlert should return an action 1`] = `
Object {
  "type": "HIDE_ALERT",
}
`;

exports[`App showAlert should return an action 1`] = `
Object {
  "message": "Alright!",
  "status": "success",
  "type": "SHOW_ALERT",
  "withTimeout": false,
}
`;
